### ISSUE TITLE
[16A] Failed test: <figcation> not matching aria-label

### ISSUE THREAD

**Issue by Hoppipollala** [Tanaguru/webextension#44]
```
<figure role="group" aria-label="Aperçu d'un champ comprenant un attribut autocomplete email.">
<img src="img/autocomplete.jpg" alt="Aperçu d'un champ comprenant un attribut autocomplete email" />
<figcaption>Aperçu d'un champ comprenant un attribut autocomplete <code>email</code>.</figcaption>
</figure>
```

Lorsque la balise `<figcaption>` contient une balise de type contenu de flux (`<code>` par exemple), la webext échoue le test, ce même si le contenu de l'attribut `aria-label` est identique et conforme.

**Comment by severineFF-tanaguru** [comment]
fix : https://github.com/Tanaguru/webextension/pull/46
